components/counter-bar/src/index.js

```diff
@@ -24,7 +24,7 @@ const TotalWrapper = styled('a')(
     outline: 'none',
     padding: '0 8px 0 0',
   },
-  ({ active }) => ({
+  ({active}) => ({
```

Comment:
minor stylistic comment - i prefer the spaces here :grin:

Reply:
Me too 😁 This rule is disabled to enable the current codebase to pass, but I'll change this one. We need to decide on our ESLint prefs!